docs/usage.md

```diff
@@ -1,47 +1,257 @@
 # nf-core/circrna: Usage
 
-## :warning: Please read this documentation on the nf-core website: [https://nf-co.re/circrna/usage](https://nf-co.re/circrna/usage)
+It is recommended that first time users run `nf-core/circrna` with the minimal test dataset either locally or on a HPC, referring to the [output documentation](https://nf-co.re/circrna/dev/output) before running a full analysis.
 
-> _Documentation of pipeline parameters is generated automatically from the pipeline schema and can no longer be found in markdown files._
-
-## Introduction
-
-<!-- TODO nf-core: Add documentation about anything specific to running your pipeline. For general topics, please point to (and add to) the main nf-core website. -->
+```console
+nextflow run nf-core/circrna -profile test,<docker/singularity/podman/institute>
+```
 
 ## Running the pipeline
```

Comment:
If you use single `#` for the main headers they will be displayed a bit more nicely on the side navigation panel on the website